src/backend/distributed/shared_library_init.c

```diff
@@ -1425,6 +1425,17 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.local_view_dependency_checks",
```

Comment:
As I noted earlier, lets make this name more generic just in case we want to use it for other objects in the future as well. Maybe `citus.enforce_object_restrictions_for_local_objects`, which is by default `true`.

Does that sound good, any alternatives?

Reply:
Also, I'm now more inclined to make this `NO_SHOW` guc. Do you have specific preference?